commands/releases/info.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+let cli  = require('heroku-cli-util');
+let co   = require('co');
+
+function* run (context, heroku) {
+  // TODO: find out how to get config vars and addons data in apiv3 or deprecate this command
+  let id = (context.args.release || 'current').toLowerCase();
+  id = id.startsWith('v') ? id.slice(1) : id;
+  let release;
+  if (id === 'current') {
+    release = (yield heroku.request({
+      path: `/apps/${context.app}/releases`,
+      partial: true,
+      headers: { 'Range': `version ..; max=1, order=desc` },
+    }))[0];
+  } else {
+    release = yield heroku.request({
+      path:    `/apps/${context.app}/releases/${id}`,
+      headers: {Accept: 'application/json'},
```

Comment:
@dickeyxxx why do you have to set this here but not [here](https://github.com/heroku/heroku-apps/pull/25/files#diff-3ced3ff9209ddc9987abea6c416f2cefR15).  I assumed it was because it is a v2 command, but you do not do that above, I guess my concern is that you may get two different formats based on the accept header here.  If that Range header is a v3 only feature, I would rather see a request to get the current, followed by a request using the response from the current as the id.